cita-jsonrpc/src/helper.rs

```diff
@@ -40,6 +40,7 @@ pub fn select_topic(method: &str) -> String {
     match method {
         "peerCount" => routing_key!(Jsonrpc >> RequestNet).into(),
         "sendRawTransaction" | "sendTransaction" => routing_key!(Jsonrpc >> RequestNewTx).into(),
+        "getVersion" => routing_key!(Jsonrpc >> RequestRpc).into(),
```

Comment:
I have some confusions, could you explain for me?
Could we `getVersion` though branch `_ => routing_key!(Jsonrpc >> Request).into(),` rather than new logic branch? 

Thinking about how we query `getBlockNumber`, `getMetaData`

Reply:
Requests will be dispatched to chain module  with routing_key!(Jsonrpc >> Request), it will be more complicated to sort out if we want to deal with it in jsonrpc module.